.changelogs/7263.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Changed the way `updateSettings` handles data buy introducing `updateData` instead of `loadData` in the majority of cases.",
```

Comment:
```suggestion
  "title": "Changed the way `updateSettings` handles data by introducing `updateData` instead of `loadData` in the majority of cases.",
```